install/openshift-web-console-operator/install.yaml

```diff
@@ -77,12 +77,12 @@ objects:
         volumes:
         - name: serving-cert
           secret:
-            defaultMode: 400
+            defaultMode: 0440
```

Comment:
Just remove the tag entirely.  This doesn't work well.

Reply:
@deads2k you mean this `serving-cert` one?